src/auth.ts

```diff
@@ -66,21 +66,19 @@ async function authorize(useLocalhost: boolean, writeToOwnKey: boolean) {
 /**
  * Loads the Apps Script API credentials for the CLI.
  * Required before every API call.
- * @param {Function} cb The callback
- * @param {boolean} isLocal If we should load local API credentials for this clasp project.
  */
-export function getAPICredentials(cb: (rc: ClaspSettings | void) => void) {
-    DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
+export async function getAPICredentials() {
+    return DOTFILE.RC_LOCAL.read().then((rc: ClaspSettings) => {
```

Comment:
One thing I want to note, is should we return the `rc` or not? It is never used, but maybe it will be in the future.

Reply:
Good comment. Had to think a bit.
If in doubt, don't expose variables/methods/values.
After `oauth2Client.setCredentials(rc);`, we don't need the credentials elsewhere so don't return the values.

In this case, remove `return rc;`
This Promise should resolve, but not return the `rc`.